x-pack/plugins/beats_management/public/pages/enforce_security.tsx

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import * as React from 'react';
+import { NoDataLayout } from '../components/layouts/no_data';
+
+export const EnforceSecurityPage: React.SFC<any> = () => (
+  <NoDataLayout title="Security Disabled" actionSection={[]}>
+    <p>
+      Security is not currently enabled on your stack. Enableing security is required for the use of
```

Comment:
Here's my first pass:

`Stack security is currently disabled. You must enable security to use Beats central management.`

@gchaps Does that sound good?

Reply:
@dedemorton, @mattapperson What about:

titile:  Security is not enabled

sentence: You must enable security to use Beats central management.

Reply:
> sentence: You must enable security to use Beats central management.

Yes, that's good. Because there's no need to restate the title in the description. :-)

Reply:
I just pushed a change that merged the two. Because security needs to be turned on for Elasticsearch and kibana I put "You must enable security in Kibana and Elasticsearch to use Beats central management." so as to be as informative as possible